### PR TITLE
chore: changed initial values for concern

### PR DESCRIPTION
**Pull request by jayanta2018**

### CODE REVIEW

File: .aws/task-definition.json [hunk 1]
```diff
@@ -21,33 +21,33 @@
...
         },
         {
           "name": "DB_USER",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-user"
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-concerns-db-user"
         },
         {
           "name": "DB_PASSWORD",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-password"
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-concerns-db-password"
```
Comment (Judge40) [inline .aws/task-definition.json:44]: This is a flaw in the template, we should copy the needed secrets from tis-hee-revalidation i.e.
```
{
          "name": "MONGODB_HOST",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-host"
        },
        {
          "name": "MONGODB_PORT",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-port"
        },
        {
          "name": "MONGODB_USERNAME",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-user"
        },
        {
          "name": "MONGODB_PASSWORD",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-password"
        }
```

File: .editorconfig [hunk 2]
```diff
@@ -13,6 +13,6 @@ trim_trailing_whitespace = false
 
 [*.java]
 # IDE specific config should not be included, but this workaround is required
-# to allow correct indentation of continued lines, otherwise an ident of two is
+# to allow correct indentation of continued lines, otherwise an indent of two is
```
Comment (Judge40) [inline .editorconfig:16]: Could you submit this fix to the template too please.

File: .aws/task-definition.json [hunk 3]
```diff
@@ -21,33 +21,33 @@
       "logConfiguration": {
         "logDriver": "awslogs",
         "options": {
-          "awslogs-group": "awslogs-tis-template",
+          "awslogs-group": "awslogs-tis-revalidation-concerns",
```
Comment (Judge40) [inline .aws/task-definition.json:24]: Make sure that this log group gets created via Terraform in TIS-OPS before the pipeline is uncommented.

File: settings.gradle [hunk 4]
```diff
@@ -1,2 +1,2 @@
 // TODO: Update project name.
```
Comment (Judge40) [inline settings.gradle:1]: Remove TODO

File: src/main/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplication.java [hunk 5]
```diff
@@ -19,16 +19,16 @@
...
  */
 
-package uk.nhs.hee.tis.template;
+package uk.nhs.hee.tis.revalidation.concerns;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 //TODO: Update package and class name.
```
Comment (Judge40) [inline src/main/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplication.java:28]: Remove TODO

File: src/main/resources/application.yml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 server:
   # TODO: Update server port and context path to a unique value.
```
Comment (Judge40) [inline src/main/resources/application.yml:2]: Remove TODO

File: src/test/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplicationTest.java [hunk 7]
```diff
@@ -19,14 +19,14 @@
...
  */
 
-package uk.nhs.hee.tis.template;
+package uk.nhs.hee.tis.revalidation.concerns;
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 //TODO: Update package and class name.
```
Comment (Judge40) [inline src/test/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplicationTest.java:28]: Remove TODO

File: .aws/task-definition.json (hunk 1 above)
Comment (jayanta2018) [reply .aws/task-definition.json:44]: Ok I will copy

File: .editorconfig (hunk 2 above)
Comment (jayanta2018) [reply .editorconfig:16]: ok sure. I will

File: src/test/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplicationTest.java (hunk 7 above)
Comment (jayanta2018) [reply src/test/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplicationTest.java:28]: Done

File: src/main/resources/application.yml (hunk 6 above)
Comment (jayanta2018) [reply src/main/resources/application.yml:2]: done

File: src/main/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplication.java (hunk 5 above)
Comment (jayanta2018) [reply src/main/java/uk/nhs/hee/tis/revalidation/concerns/ConcernsApplication.java:28]: done

File: settings.gradle (hunk 4 above)
Comment (jayanta2018) [reply settings.gradle:1]: done

File: .aws/task-definition.json (hunk 3 above)
Comment (jayanta2018) [reply .aws/task-definition.json:24]: ok I will

File: .aws/task-definition.json [hunk 8]
```diff
@@ -21,33 +21,53 @@
...
+          "name": "RABBITMQ_PORT",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-rabbit-port"
+        },
+        {
+          "name": "RABBITMQ_USERNAME",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-rabbit-username"
+        },
+        {
+          "name": "TCS_URL",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tcs-url"
```
Comment (Judge40) [inline .aws/task-definition.json:64]: Do we need Rabbit and TCS for concerns?

Comment (jayanta2018) [reply .aws/task-definition.json]: @Judge40 I have removed tcs and rabbit secrets

**Comment by jayanta2018** [comment]
> I think project name should be TIS-HEE-CONCERN. It should not be revalidation. Concern is a separate business case.

@uzairbaig We have discussed it over the call just now

**Pull request merged**